website/static/js/fangorn.js

```diff
@@ -473,9 +476,9 @@ function checkConflicts(items, folder){
 }
 
 function handleCancel(tb, provider, mode, item){
+    tb.modal.dismiss();
     if (mode === 'stop') {
         tb.syncFileMoveCache[provider].conflicts.length = 0;
-        tb.modal.dismiss();
```

Comment:
can't figure out what this line is doing!

Reply:
I think this is just a "bug" it's usually just not expressed because the modal is dismissed elsewhere.